hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/util/ConsistentHashingUpdateStrategyUtils.java

```diff
@@ -83,7 +83,7 @@ private static void extractHashingMetadataFromClusteringPlan(String instant, Hoo
       ValidationUtils.checkState(p != null, "Clustering plan does not has partition info, plan: " + plan);
       // Skip unrelated clustering group
       if (!recordPartitions.contains(p)) {
-        return;
+        continue;
```

Comment:
would you please clarify why the logic changed here

Reply:
It's a bug in the origin code.  The set `recordPartitions` contains all partitions that we care about, and we iterate over input groups in clustering plan to construct hashing metadata. If current input group is not included in `recordPartitions` we need to continue to check the next input group instead of return directly.